alerts/cluster-network-operator/NorthboundStaleAlert

```diff
@@ -0,0 +1,56 @@
+  
+   You can check the containers running on the ovn-kube master nodes with this command 
+   `$oc get pods -n=openshift-ovn-kubernetes -o jsonpath='{range .items[*]}{"\n"}{.metadata.name}{":\t"}{range .spec.containers[*]}{.name}{", "}{end}{end}' |sort | grep master`
+   You should also run `$oc get pods -n=openshift-ovn-kubernetes | grep master` to ensure that all containers on your master pods are alive
+   To see the pod logs you can run `$oc describe pod/<podname> -n <namespace>`
+   To see the logs of a specific container within the pod you can run `$oc logs <podname> -n <namespace> -c <containerName>`
+
+2. Is OVN NBDB functioning. Check to see if the NBDB containers are running without errors. If there are no errors, ensure there is an OVN NBDB Leader.
+   
+   If you have made it through the debug steps in step one, then you should already know that the containers are healthy on the master pod/s, and you should now ensure 
+   that there is an NBDB leader. 
+   This command will show you what master pod the leaders are on (The NBDB leader and SDBD Leader should be on the same pod).
+   `$oc -n openshift-ovn-kubernetes get po -l app=ovnkube-master --field-selector spec.nodeName=$(oc -n openshift-ovn-kubernetes get cm ovn-kubernetes-master -o jsonpath='{.metadata.annotations.control-plane\.alpha\.kubernetes\.io\/leader}' | jq -r '.holderIdentity') -o jsonpath='{.items[0].metadata.name}'`
+   A common cause of db leader issues is that one of the database servers is unable to participate with the other raft peers due to mismatching cluster ids. Due to this mismatch, they will be unable to elect a db leader. Wiping the dbs and restarting the master pods typically resolves this issue. 
+   
+
+3. Lastly, check to make sure that the connectivity between ovnkube-master leader and OVN NBDB leader is healthy.
+   To determine what node the ovnkube-master is on, run `oc get cm -n openshift-ovn-kubernetes ovn-kubernetes-master -o json | jq '.metadata.annotations'`
+   Then get the logs of the ovnkube-master container in the ovnkube-master pod on that node with `$oc logs <podname> -n <namespace> -c ovnkube-master`
+   You should see an message along the lines of "msg"="trying to connect" "database"="OVN_Northbound" "endpoint"="tcp:172.18.0.4:6641". This message indicates that the master cannot connect to the database
+   A successful connection message will appear in the logs if the master has connected to the database
+
+
+ 
+## Mitigation 
+
+Mitigation will depend on what was found in the diagnosis section. 
+Typically, deleting the dbs and then restarting the master pods will take care of most issues.
+You can use the following commands to accomplish this.
+
+for master_node in $(oc get nodes --selector="node-role.kubernetes.io/master"="" -o jsonpath='{range .items[*].metadata}{.name}{"\n"}{end}'); do \
+  oc debug node/${master_node} -- chroot /host sh \
+   -c 'rm -f /var/lib/ovn/etc/ovn*_db.db
```

Comment:
Just to confirm, will it have any side-effect to delete the db?

Reply:
The dbs have the ability to repopulate themselves in ovn, so afik there are not side effects to this other than cleaning out stale connections

Reply:
@flavio-fernandes knows more about this than i do but what i said is the tldr